deployers/whiskclient.go

```diff
@@ -291,5 +308,11 @@ func validateClientConfig(credential PropertyValue, apiHost PropertyValue, names
 		wskprint.PrintOpenWhiskVerbose(utils.Flags.Verbose, stdout)
 	}
 
+	if len(apigwTenantId.Value) != 0 {
```

Comment:
Since this is verbose, should the user know that their APIGW_TENANT_ID is empty if they are targeting CF namespace? I think we should print out the apigwAccessTenantId regardless the type of namespace being targeted.

Reply:
It is my belief that as the code is not authored that all values are either printed (verbose) or not (default/non-verbose); there should be no distinction between these 2 values or any correlation between them that assumes one or the other (i.e., should always print all known properties without control logic).

Reply:
ok good to know.